server/pages/index.js

```diff
@@ -143,7 +143,13 @@ function getDefaultContext( request ) {
 		isFluidWidth: !! config.isEnabled( 'fluid-width' ),
 		abTestHelper: !! config.isEnabled( 'dev/test-helper' ),
 		devDocsURL: '/devdocs',
-		store: createReduxStore( initialServerState )
+		store: createReduxStore( initialServerState ),
+		css: new Set(),
```

Comment:
a set doesn't maintain order, while CSS can be order dependent. Does that matter, or are we assuming that our CSS is not order dependent?